skbio/io/format/embl.py

```diff
@@ -0,0 +1,1497 @@
+                   'DE': 'DEFINITION',
+                   'GN': 'GENE_NAME',  # uniprot specific
+                   'KW': 'KEYWORDS',
+                   # Source (taxonomy and classification)
+                   'OS': 'ORGANISM',
+                   'OC': 'taxonomy',
+                   'OG': 'organelle',
+                   # reference keys
+                   'RA': 'AUTHORS',
+                   'RP': 'REFERENCE',
+                   'RC': 'REFERENCE_COMMENT',
+                   'RX': 'CROSS_REFERENCE',
+                   'RG': 'GROUP',
+                   'RT': 'TITLE',
+                   'RL': 'JOURNAL',
+                   # Cross references
+                   'DR': 'DBSOURCE',
+                   'CC': 'COMMENT',
+                   # features
+                   'FH': 'FEATURES',
+                   'FT': 'FEATURES',
+                   'SQ': 'ORIGIN',
+                   }
+
+# the original genbank _yield_section divides entries in sections relying on
+# spaces (the same section has the same level of indentation). Uniprot entries
+# have a key for each line, so to divide record in sections I need to define a
+# corresponance for each key to section, then I will divide a record in
+# sections using these section name. Some keys are commented out since I don't
+# find them in example. What I have to do if I find them?
+KEYS_2_SECTIONS = {
+                   # identification
+                   'ID': 'LOCUS',
+                   'AC': 'ACCESSION',
+                   # PA means PARENT ACCESSION (?) and applies to
+                   # feature-level-products entries
+                   'PA': 'PARENT_ACCESSION',
+                   'PR': 'PROJECT_IDENTIFIER',
+                   'DT': 'DATE',
+                   'DE': 'DEFINITION',
+                   'GN': 'GENE_NAME',  # uniprot specific
+                   'KW': 'KEYWORDS',
+                   # Source (taxonomy and classification)
+                   'OS': 'SOURCE',
+                   'OC': 'SOURCE',
+                   'OG': 'SOURCE',
+                   # reference keys
+                   'RA': 'REFERENCE',
+                   'RP': 'REFERENCE',
+                   'RC': 'REFERENCE',
+                   'RX': 'REFERENCE',
+                   'RG': 'REFERENCE',
+                   'RT': 'REFERENCE',
+                   'RL': 'REFERENCE',
+                   'RN': 'SPACER',
+                   # Cross references
+                   'DR': 'DBSOURCE',
+                   'CC': 'COMMENT',
+                   'AH': 'ASSEMBLY',
+                   'AS': 'ASSEMBLY',
+                   'FH': 'FEATURES',
+                   'FT': 'FEATURES',
+                   # sequence
+                   'SQ': 'ORIGIN',
+                   '  ': 'ORIGIN',
+                   # 'CO': 'ORIGIN,'
+                   # spacer (discarded)
+                   'XX': 'SPACER'
+                  }
+
+
+# for convenience: I think such functions are more readadble whiel accessing
```

Comment:
I've always been on the fence with lambdas here. The justification can be found [here](https://www.python.org/dev/peps/pep-0008/). Part of the motivation is improved clarity on tracebacks.